### PR TITLE
Update time and home page after exiting screensaver

### LINKED ISSUE #193: Time doesn't update after exiting screensaver
**Describe the bug**
The time on the overhang doesn't get refreshed after exiting the screensaver. Although it does continue to get updated every minute as expected.

Opening any new screen or going back a screen will fix the time.

**To Reproduce**
1. While on a screen showing the time, let the screen saver kick in.
2. Let the screensaver run for at least a few minutes
3. Press a button on the remote to deactivate the screensaver
4. Observe the time

**Expected behavior**
Time gets refreshed

**Additional context**

### PR DESCRIPTION
**Pull request by cewert**
**Changes**
Use EnableScreensaverExitedEvent() to be notified when user exits screensaver
reset the clock after exiting screensaver
refresh the home page after exiting screensaver if home page is active

**Issues**
Fixes #193

### CODE REVIEW

File: source/Main.brs [hunk 1]
```diff
@@ -364,6 +368,14 @@ sub Main()
         'Event will be called on caption change which includes the current mute status, but we do not want to call until the overlay is closed
           reviewSubtitleDisplay()
         end if
+      else if event.exitedScreensaver = true then
+        m.overhang.callFunc("resetTime")
+        if group.subtype() = "Home" then
+          currentTime = CreateObject("roDateTime").AsSeconds()
+          group.timeLastRefresh = currentTime
```
Comment (neilsb) [inline source/Main.brs:375]: Not directly related to this, but may be worth having the Refresh function update the _LastRefreshTime_ to save having to remember to do it in all the places the function is called from.   Could even have the Refresh function compare the times to decide whether to perform the refresh or just return.

Comment (neilsb) [reply source/Main.brs:375]: Ignore that - I'll raise another issue since it's not related to this.  Sorry.

**Review by neilsb — APPROVED** [review]
All seems to work as intended.  👍

File: source/Main.brs (hunk 1 above)
Comment (cewert) [reply source/Main.brs:375]: Great idea. Yea let's put that in it's own PR

**Review by anthonylavado — APPROVED** [review]

**Comment by anthonylavado** [comment]
@neilsb I need to remember to invite you to the team, if you're still open to it :-)

It'll make those review checkmarks green

**Pull request merged**